fizzbuzz/Python/RansomBowman/unitTests.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from FizzBuzz import fizzBuzz
+
+class TestFizzBuzz(unittest.TestCase):
+        
+    def test_3(self):
+        self.assertEqual(fizzBuzz(3), fizzBuzz(3))
```

Comment:
@RansomBowman - in this case you are asserting the function call against the same function call.  Instead have your assertion be against the value you'd expect the function to return:
```self.assertEqual(fizzBuzz(3), "Fizz")```